### DISCUSSION TITLE
Grey Demo Dashboard Cutoff

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#912] [General]
![image](https://user-images.githubusercontent.com/4793561/125144108-b1296980-e0d1-11eb-860e-d71a2be17f47.png)

Seeing this for Wave 0.17.0, only the Grey dashboard, no others. This only happens in Safari, not chrome. On both my 15" MacBook and 26" external monitor. I have not tried debugging yet but wanted to share this safari-incompatibility

**Comment by mturoci** [comment]
Tracked in https://github.com/h2oai/wave/issues/939
